src/main/java/fi/aalto/cs/intellij/notifications/Notifier.java

```diff
@@ -0,0 +1,10 @@
+package fi.aalto.cs.intellij.notifications;
+
+import com.intellij.notification.Notification;
+import com.intellij.openapi.project.Project;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
```

Comment:
would you feel OK to mark functional interfaces as such, as it would improve the readability: https://docs.oracle.com/javase/8/docs/api/java/lang/FunctionalInterface.html

Reply:
Wasn't aware of that. Certainly!